test.js

```diff
@@ -1,27 +1,20 @@
-'use strict';
-var test = require('ava');
-var ghUser = require('./');
+import test from 'ava';
+import fn from './';
 
-test('user exists', function (t) {
-	return ghUser('sindresorhus').then(function (user) {
-		t.is(user.login, 'sindresorhus');
-	});
+test('user exists', async t => {
+	const user = await fn('sindresorhus');
+	
+	t.is(user.login, 'sindresorhus');
 });
 
-test('user doesn\'t exist', function (t) {
-	return ghUser('sindr555esorhus').catch(function (err) {
-		t.is(err.message, 'Response code 404 (Not Found)');
-	});
+test('user doesn\'t exist', async t => {
+	await t.throws(fn('sindresorhus'), Error('Response code 404 (Not Found)'));
 });
 
-test('no username given (no arg)', function (t) {
-	return ghUser().catch(function (err) {
-		t.is(err.message, '`username` required');
-	});
+test('no username given (no arg)', async t => {
+	await t.throws(fn(), /`username` required/);
```

Comment:
The only way I can get these tests to work is by providing a regex.

These tests succeed
```js
test('no username given (no arg)', async t => {
	await t.throws(fn(), 'foo bar');
});

test('no username given (empty string)', async t => {
	await t.throws(fn(''), 'foo bar');
});
```

According to the [docs](https://nodejs.org/api/assert.html#assert_assert_throws_block_error_message) it doesn't look like the second argument can be a string. So I tried other things like this as well

```js
test('no username given (no arg)', async t => {
	await t.throws(fn(), Error, 'foo bar');
});

test('no username given (empty string)', async t => {
	await t.throws(fn(''), Error, 'foo bar');
});
```

But the tests still succeed. The only way I can get this to work correctly is by using a regex.

Tried to have a look at the AVA source code, the `throws` implementation is not that hard but I can't find what's wrong here. Might be related to my `pageres-cli` test issues with throws.